sdk/src/integrationTest/resources/raw/offers_get_similar_brand_spending.json

```diff
@@ -0,0 +1,7 @@
+{
+  "data": [],
```

Comment:
Need real response here.

Reply:
I'll ask for some json from banana stand